cpp/src/plasma/store.cc

```diff
@@ -411,6 +411,39 @@ int PlasmaStore::abort_object(const ObjectID& object_id, Client* client) {
   }
 }
 
+int PlasmaStore::delete_object(ObjectID& object_id) {
```

Comment:
Given that we already have a `delete_objects` method, does it make sense to also have a `delete_object` method? Is the behavior of this method different from calling `delete_objects` on a vector of one object ID?

Reply:
To be honest, they are similar. But in delete_object will issue the error status to client if delete object failed. As for delete_objects used by evict objects function, I don't want to impact it too much.